CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.9.0](https://github.com/sonata-project/SonataNewsBundle/compare/3.8.0...3.9.0) - 2019-09-02
+### Added
+- Add missing `twig/twig` required in `composer.json` with versions `^1.35 || ^2.4`
+- Added events to the comment process
+- Added a new test class SonataNewsExtensionTest. Inspired by the
+  SonataMediaExtensionTest, I extended AbstractExtensionTestCase to test if the
+association with media, user, tag, collection have been created according to a
+configuration
+- A dependency to [matthiasnoback/symfony-dependency-injection-test
+](https://packagist.org/packages/matthiasnoback/symfony-dependency-injection-test)
+- Added support for flex
```

Comment:
Not yet. We need a recipe which is not possible for now :(

Reply:
Do I just remove that line then?